shoal/tool/src/java/uk/ac/ox/it/shoal/Tuple.java

```diff
@@ -0,0 +1,18 @@
+package uk.ac.ox.it.shoal;
+
+/**
+ * Simpe tuple.
+ * @author buckett
+ *
+ * @param <X>
+ * @param <Y>
+ */
+public class Tuple<X, Y> {
+	public final X x;
```

Comment:
You still need the subclass and I don't need the extra functionality.
But thanks for the pointer.